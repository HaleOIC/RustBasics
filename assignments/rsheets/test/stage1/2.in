set B2 10
get B2
set F6 sleep_then(500, 15)
get F6