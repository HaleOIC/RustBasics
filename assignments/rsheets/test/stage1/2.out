B2 = 10
F6 = 15